edx_proctoring/templates/proctoring/seq_proctored_exam_error.html

```diff
@@ -0,0 +1,56 @@
+{% load i18n %}
+<div class="failure sequence proctored-exam" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      Your proctoring session is in error
+    {% endblocktrans %}
+  </h3>
+
+  <h4>
+    {% blocktrans %}
+      Your Proctoring Session review: <b class="failure"> Failed </b>
```

Comment:
I think we are going with "Unsatisfactory" and "Satisfactory"
@chrisndodge @griffresch